### PR TITLE
Source Control Challenge - Marom Goldshmidt

### PR DESCRIPTION
**Pull request by MaromGold**
**2nd week challenge:** 
This bootstrap script installs Git, configures it, and installs Docker and runs a Jenkins container.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
 tar -zxf git.tar.gz
+# The next line cds to the last created folder, whuch is the extracted files' folder
 cd  "$(\ls -1dt ./*/ | head -n 1)"
-make configure
 ./configure --prefix=/usr/local
-sudo make install
+make install
+
+#Git initial setup:
+git config --global user.name "Marom Goldshmidt"
```
Comment (noamgu) [inline bootstrap.sh]: Hard coded values are meh.
Accept them as arguments.
Make sure you are getting them from the user, else raise exception.

Comment (MaromGold) [reply bootstrap.sh]:
```suggestion
git config --global user.name $1
```

**Comment by MaromGold** [comment]
@noamgu Please review so I can submit a new PR

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**